eternalcore-api-example/src/main/java/com/eternalcode/example/feature/home/ApiHomeCommand.java

```diff
@@ -0,0 +1,53 @@
+package com.eternalcode.example.feature.home;
+
+import com.eternalcode.core.feature.home.HomeService;
+import dev.rollczi.litecommands.annotations.argument.Arg;
+import dev.rollczi.litecommands.annotations.command.Command;
+import dev.rollczi.litecommands.annotations.context.Context;
+import dev.rollczi.litecommands.annotations.execute.Execute;
+import org.bukkit.Location;
+import org.bukkit.entity.Player;
+
+@Command(name = "api home")
+public class ApiHomeCommand {
+
+    private final HomeService homeService;
+
+    public ApiHomeCommand(HomeService homeService) {
+        this.homeService = homeService;
+    }
+
+    @Execute(name = "set")
+    void executeSet(@Context Player player, @Arg("homeName") String homeName) {
+        this.homeService.createHome(player.getUniqueId(), homeName, player.getLocation());
+        player.sendMessage("Home set!");
```

Comment:
why are we not using noticeService. I get that it's API, but why...

Reply:
because it's an example plugin, noticeService is meant for internal EternalCore usage